packages/bot-web-ui/src/components/audio/__tests__/audio.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Audio from '../audio';
+
+describe('Audio', () => {
+    it('should render Audio', () => {
+        const { container } = render(<Audio />);
+        expect(container).toBeInTheDocument();
+    });
+
+    it('should have 5 audio tag', () => {
+        render(<Audio />);
+        expect(screen.getAllByLabelText('audio')).toHaveLength(5);
+    });
+});
```

Comment:
@vinu-deriv Bhai can we also check if the audio elements have the src attributes as it is a mandatory one?